CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ All notable changes to the "vscode-java-debugger" extension will be documented i
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.12.0 - 2018-8-29
+### Added
+- Start without debugging. See [PR@198](https://github.com/Microsoft/java-debug/pull/198),[PR#204](https://github.com/Microsoft/java-debug/pull/204), [PR@207](https://github.com/Microsoft/java-debug/pull/207)
```

Comment:
Somewhere you use PR@ and PR#, please keep consistent. Use PR# pattern.

Reply:
Please link to the issue https://github.com/Microsoft/vscode-java-debug/issues/351, it has more explanation about this feature.